dist/preview release/what's new.md

```diff
@@ -3,6 +3,9 @@
 ## Major updates
 
 ## Updates
+- Add support for specifying the center of rotation to textures ([bghgary](http://www.github.com/bghgary))
+- Add support for KHR_texture_transform to glTF 2.0 loader ([bghgary](http://www.github.com/bghgary))

```

Comment:
Do you mind creating a gltf section like I did for previous what's new
I guess we will have more to come:)